packages/fxa-auth-server/lib/senders/emails/partials/automatedEmailResetPassword/index.mjml

```diff
@@ -1,15 +1,21 @@
-<%# This Source Code Form is subject to the terms of the Mozilla Public
-  # License, v. 2.0. If a copy of the MPL was not distributed with this
-  # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+<%# This Source Code Form is subject to the terms of the Mozilla Public #
+License, v. 2.0. If a copy of the MPL was not distributed with this # file, You
+can obtain one at http://mozilla.org/MPL/2.0/. %>
 
 <mj-section>
   <mj-column>
     <mj-text css-class="text-footer-automatedEmail">
       <span data-l10n-id="automated-email-reset">
         This is an automated email; if you did not authorize this action, then
-        <a class="link-blue" href="<%- resetLink %>" data-l10n-name="resetLink">please reset your password</a>.
-        For more information, please visit
-        <a class="link-blue" href="<%- supportUrl %>" data-l10n-name="supportLink">Mozilla Support</a>.
+        <a class="link-blue" href="<%- resetLink %>" data-l10n-name="resetLink"
+          >please reset your password</a
+        >. For more information, please visit
```

Comment:
@jodyheavener We should probably get prettier figured out for these files sooner than later, maybe in a follow up we can mass format the MJML files we have so far. This seems to be how it default formats HTML files but I'm sure there's rules we can set because I also don't particularly like this 😅

Reply:
Yeah agree, would you be okay with just not formatting MJML files for now?

Reply:
Sure, so long as you don't mind if I temporarily add `*.mjml` to the `.prettierignore` file. I'm not sure why yours was not formatting without sticking it in there 🤔